sandy/decay.py

```diff
@@ -230,6 +237,78 @@ def get_br(self, with_uncertainty=True):
         else:
             return df.BR
 
+    def get_decay_energy(self, with_uncertainty=True):
```

Comment:
Here you use underscores, while in `get_decay_constants` you don't. Then in `get_br` you use initials. Choose one and be consistent! This is the one I prefer

Reply:
By the way, I guess switching `get_decayconstant` to `get_half_lives` will facilitate the pipeline. What do you think?

Reply:
Indeed we will use the already implemented  `get_half_life` method to extract the BE information so I will remove `get_decay_constant` and sobstitute  the `custom_perturbation_decayconstant` method with the one for half life in another pull request